package.json

```diff
@@ -0,0 +1,67 @@
+{
+  "name": "TEAM-A",
+  "version": "1.0.0",
+  "description": "This is our Readme file. Let's build something!",
+  "main": "main.js",
+  "scripts": {
+    "start-dev": "npm run build-watch & npm run start-server",
+    "start-server": "nodemon main.js -e html,js,css --ignore public",
+    "build-watch": "webpack -w",
+    "start": "node main.js",
+    "seed": "node seed.js"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/1909-FLEX-GRACE-SHOPPER/TEAM-A.git"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/1909-FLEX-GRACE-SHOPPER/TEAM-A/issues"
+  },
+  "homepage": "https://github.com/1909-FLEX-GRACE-SHOPPER/TEAM-A#readme",
+  "dependencies": {
+    "@date-io/date-fns": "^2.0.1",
+    "@material-ui/core": "^4.9.0",
+    "@material-ui/icons": "^4.5.1",
+    "@material-ui/pickers": "^3.2.9",
+    "@material-ui/styles": "^4.9.0",
+    "axios": "^0.19.1",
+    "babel": "^6.23.0",
+    "babel-cli": "^6.26.0",
+    "babel-preset-env": "^1.7.0",
+    "babel-preset-react": "^6.24.1",
+    "bcrypt": "^3.0.7",
+    "chalk": "^3.0.0",
+    "cjs": "0.0.11",
+    "cookie-parser": "^1.4.4",
+    "date-fns": "^2.9.0",
+    "express": "^4.17.1",
+    "faker": "^4.1.0",
+    "nodemon": "^2.0.2",
+    "npm-run-all": "^4.1.5",
+    "path": "^0.12.7",
+    "pg": "^7.17.1",
+    "react": "^16.12.0",
+    "react-dom": "^16.12.0",
+    "react-redux": "^7.1.3",
+    "react-router-dom": "^5.1.2",
+    "redux": "^4.0.5",
+    "redux-devtools-extension": "^2.13.8",
```

Comment:
dev-dep. HOWEVER -- check how you are actually implementing in your code. You'll probably need to check for a development mode then use this and other dev tools